README.md

```diff
@@ -6,5 +6,6 @@ Django Backend for attempting to pick the entire SEC football schedule before th
 - ~~setup docker and docker compose~~
 - ~~setup postgres instead of sqlite~~
 - ~~setup secret key and database with [docker secrets](https://docs.docker.com/engine/swarm/secrets/)~~
-- setup database fixture for Teams
-- setup unit tests with [testcontainers](https://testcontainers.com)
+- ~~setup database fixture for Teams~~
+- Setup ability to run unit tests individually using the UI in Pycharm
```

Comment:
Not sure why this doesn't work yet, but I think that's the main thing stopping me from continuing on this
<img width="407" alt="Screenshot 2024-04-24 at 4 02 48 PM" src="https://github.com/andmehta/cfb-picker-backend/assets/38476199/b41b62e9-5776-4a4f-9962-87efc49e286a">

Reply:
eehhhh who cares? Lol